core/modules/proto/testProtocol.cc

```diff
@@ -172,7 +173,7 @@ BOOST_AUTO_TEST_CASE(MsgBuffer) {
 BOOST_AUTO_TEST_CASE(ProtoHashDigest) {
     std::unique_ptr<lsst::qserv::proto::TaskMsg> t1(makeTaskMsg());
     std::string hash = hashTaskMsg(*t1);
-    std::string expected = "f3890e99c399ab196c164eccf2e99369";
+    std::string expected = "b3ecc6a56205c90642e61943e44c7f14";
```

Comment:
This is not a good test, in my opinion. The hash changes every time the protocol changes and needs to be replaced. I'm not sure what a better test is. Perhaps check that running the hash twice for t1 produces the same result, and does not equal the hash of a slightly different TaskMsg?